api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -42,11 +42,81 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 ### Data Dictonary [GET]
 
 + Response 200 (application/json)
-    + Attributes 
+    + Attributes
         + document (DataDictionary)
-    
+
+
+## Download Count [/api/v2/download/count/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Download Count Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (required, object)
+            + time_period (array[TimeRangeObject])
+
++ Response 200 (application/json)
+    + Attributes
+        + transaction_rows_gt_limit : true (required, boolean)
+            A boolean returning whether the transaction count is over the maximum row limit.
+
+## Reference Filter [/api/v1/references/filter/]
```

Comment:
This filter also belongs in the Advanced Search file, and should be renamed to something more descriptive like `Generate URL Hash`.